training/platform-services/DEPLOYMENT_AND_RUNNING.md

```diff
@@ -0,0 +1,109 @@
+Deployment and Running
+===========================
+
+## Concourse
+
+Visit <https://concourse.onsdigital.co.uk> to access Concourse
+
+[<img src="images/concourse-home.png" title="Concourse Homepage" alt="concourse ons" width=49.7% height=49.7%/>][concourse-ons] [<img src="images/concourse-pipeline.png" title="Concourse Pipeline" alt="concourse pipeline" width=49.7% height=49.7%/>][concourse-ons]
+
+Concourse lists all of the applications where each application has its own [pipeline][concourse-pipeline].
+When a [PR][git-pr] of an application is said to be merged into `develop` or `master`/`main`, Concourse runs the pipeline of the application which does unit testing, checks if a [docker image][docker-image] can be built for it, bundles it for release and ships it to that release whilst ensuring that it has deployed everything completely.
+
+Furthermore, Concourse helps us to identify if an application has been fully deployed with any new changes implemented and can provide information on the version of the application which is currently running in the `develop` and `production` environments.
+
+Please note that when deploying changes to the `production` environment (the live website), you need to **manually** deploy the application by doing the following:
+
+1. Go to [Concourse][concourse-ons]
+2. Navigate to the application
+3. Click on `production-ship-it`
+4. <div>Click on the <img src="images/concourse-build.png" title="Concourse Build" width=4% height=4%></div>
+5. Login to concourse as `dp` user (if you do not have the login, please ask anyone in the team)
```

Comment:
Should this step come before some of the above ?

Reply:
This has now been amended